include/NAS2D/StringUtils.h

```diff
@@ -1,6 +1,6 @@
 // ==================================================================================
 // = NAS2D
-// = Copyright � 2008 - 2019 New Age Software
+// = Copyright � 2008 - 2019 New Age Software

```

Comment:
Small nit happened here. Consider changing to non-unicode string of three characters: `(c)`.

Reply:
Gah! I hate when that happen. Atom seems to silently and automatically mess with certain encoded symbols.

So it looks like the © was ASCII encoded, rather than Unicode encoded. I believe other files in the project do similar. Perhaps we should switch to Unicode encoding. I'll open a new issue to track this.